jupyter_notebook_parser/container.pyi

```diff
@@ -0,0 +1,23 @@
+import sys
+
+if sys.version_info < (3, 10):
+    from typing import Tuple, List, Dict
+else:
+    List = list
+    Tuple = tuple
+    Dict = dict
```

Comment:
This renaming is totally unnecessary.  Upper case type names are compatible in Python 3.10+

Reply:
Aliases from typing for built-in containers are deprecated since 3.9. Checking 3.10 reuired to prevent conficts in 3.9 beta versions

Reply:
Upper case type names are still compatible with Python 3.10+